pkg/cmd/server/api/types.go

```diff
@@ -980,3 +983,19 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// PodLimitRequestConfig is the configuration for the PodLimitRequest
+// admission controller which overrides user-provided container request/limit values.
+type PodLimitRequestConfig struct {
```

Comment:
This name doesn't describe what this is for - I'd prefer more useful information.  Something like PodOvercommitPolicy or PodRequestLimitOverride or ClusterOvercommitPolicy (which describes the intent of the config, not the mechanics).  If like the admission controller name to match the intent as well.  I'm open to other suggestions, but I don't think PodRequestLimit is enough.